.github/workflows/pr-automation-auto-assign.yml

```diff
@@ -0,0 +1,12 @@
+name: PR Automation
```

Comment:
This change is out of context.

Reply:
File Removed
https://github.com/schul-cloud/node-notification-service/pull/205/commits/b0dfe1eba549fded8b9ebd7fe7d6e39bb5798f15